test/e2e/tests/custom-rpc-history.spec.js

```diff
@@ -49,7 +49,7 @@ describe('Stores custom RPC history', function () {
         await chainIdInput.sendKeys(chainId.toString());
 
         await driver.clickElement('.network-form__footer .btn-secondary');
-        await driver.findElement({ text: networkName, tag: 'div' });
+        await driver.findElement({ text: networkName, tag: 'span' });
```

Comment:
Do we know if and/or how this test passing before? I'm not seeing a change to the HTML here.

Reply:
Its because before this change the networks setting page was shown and so now its just finding the network name in a completely different place: 
It was finding it here before:
<img width="706" alt="Screen Shot 2021-09-03 at 10 50 56 AM" src="https://user-images.githubusercontent.com/34557516/132033335-47ac1e60-38ee-4ebf-837a-5e948b2705b3.png">
And here now:
<img width="193" alt="Screen Shot 2021-09-03 at 10 50 26 AM" src="https://user-images.githubusercontent.com/34557516/132033243-fb84965f-08d1-411a-a158-9d417f15f68c.png">